src/handler/mod.rs

```diff
@@ -468,6 +485,27 @@ impl Stream for Handler {
                     HandlerMessage::DisposeContext(ctx) => {
                         pin.browser_contexts.remove(&ctx);
                     }
+                    HandlerMessage::AddEventListener(req) => {
+                        pin.event_listeners.add_listener(req);
+                    }
+                    HandlerMessage::GetPage(target_id, tx) => {
+                        let target = pin
+                            .targets
+                            .values_mut()
+                            .find(|p| p.target_id() == &target_id);
+
+                        match target {
```

Comment:
This can be simplified as something like this
`let page = self.targets.get_mut(id).and_then(|target| {
                            target.get_or_create_page()
                        }).map(|page| Page::from(page.clone()));`